main.py

```diff
@@ -1,10 +1,75 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
+PLAYERS = "players.json"
+
+
+def load_data_json():
+    with open(PLAYERS) as file:
+        return json.load(file)
+
+
+def get_race_info():
+    return [value["race"] for key, value in load_data_json().items()]
+
+
+def create_race_and_skill_model():
+    """
+    This function making Skill and Race models
+    """
+    data = get_race_info()
+    for race in data:
+        if not Race.objects.filter(name=race["name"]).exists():
+            Race.objects.create(name=f"{race['name']}",
+                                description=f"{race['description']}")
+
+        for skill in race["skills"]:
+            if not Skill.objects.filter(name=skill["name"]).exists():
+                Skill.objects.create(name=skill['name'],
+                                     bonus=skill['bonus'],
+                                     race=Race.objects.get(
+                                         name=f"{race['name']}"))
+
+
+def create_guild_model():
+    """
+    This function making Guild models
+    """
+    data = load_data_json()
+    guilds_data = [value["guild"] for key, value in data.items()]
+    for guild in guilds_data:
+        if guild is not None:
+            if not Guild.objects.filter(name=guild["name"]).exists():
+                if guild["description"] is not None:
```

Comment:
you can omit this check. When you create a `guild` description will be set to some value or None

Reply:
Thanks! I fixed all problems